BeeSwift/JSONGoal.swift

```diff
@@ -633,7 +633,7 @@ class JSONGoal {
             
             if datapoints.count == 0 {
                 let requestId = "\(daystamp)-\(self.minuteStamp())"
-                let params = ["access_token": CurrentUserManager.sharedManager.accessToken!, "urtext": "\(daystamp.suffix(2)) \(datapointValue) \"Automatically entered via iOS Health app\"", "requestid": requestId]
+                let params = ["access_token": CurrentUserManager.sharedManager.accessToken!, "urtext": "\(daystamp.suffix(2)) \(datapointValue) \"Automatically entered via Apple Health app\"", "requestid": requestId]
```

Comment:
I personally would prefer "via Apple Health" or "via the Apple Health app" (using the and app together).

Reply:
Furthermore, not part of the ticket mentioned but part of the lines adjusted, all other autodata goals I have show 'Auto-entered' and 'Auto-updated'.
I am not sure if it is a beeminder standard or just happened that way because all other autodata goals take place on the backend directly.
How about changing that as well to match the others? 

@dreeves Would you care to standardize on the text for datapoint comments when created and updated using an autodata source?

Reply:
I propose: \"Automatically entered via Apple Health\" -- no "the", no "app"

Reply:
That tackles the 'Apple Health' part.

And what about the automatically entered vs auto-entered (as is the case with autodata goals)?

Reply:
Oh, right, yes, let's get that consistent! Here's an example from my GmailZero goal: "Auto-entered by gmailzero at 17:01. Last count 109 at 09:30:06". Can we model it on that? "Auto-entered by Apple Health at ___". Not sure if there's other useful data like that that could be included in the comment.